### PR TITLE
Yyuan/neg sample study

### PR DESCRIPTION
**Pull request by yiweny**

### CODE REVIEW

File: run_efficiency.sh
Comment (rishabh-ranjan) [inline run_efficiency.sh:1]: maybe this should be in examples?

**Review by rishabh-ranjan — CHANGES_REQUESTED** [review]
Can you describe this PR please? Thanks

File: run_efficiency.sh
Comment (yiweny) [reply run_efficiency.sh:1]: I am trying to support sampling pos/negs with different ratios. But I hit some technical difficulties.

Comment (yiweny) [reply run_efficiency.sh:1]: Ohh. my b

**Pull request closed without merging**